### PR TITLE
Dcs multi factor authentication

### PR DESCRIPTION
**Pull request by DCorneliusS**
### Description

The documentation entry "multi-factor-authentication.md" found in docs/content/cybersecurity/concepts/multi-factor-authentication has been edited. All five of the possible authentication factors have been added to the entry alongside several examples for each type of factors. Common two-factor authentication (2FA) methods have also been touched upon given the recent rise in the use and mention of 2FA.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@DCorneliusS Looks good for a second review as is.

File: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md [hunk 1]
```diff
@@ -14,21 +14,72 @@ CatalogContent:
 
 **Multi-factor Authentication (MFA)** is the use of more than one factor to prove the identity of a user.
 
+One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require users to create when initially signing up for an account.
```
Comment (yangc95) [inline content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md:17]: Can we create a segue to the other factors below

File: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md [hunk 2]
```diff
@@ -14,21 +14,72 @@ CatalogContent:
...
+
 ## Factors
 
-One factor is typically a username or email and password combination. This is the basic log-in functionality that most websites require users to create when initially signing up for an account.
+There are, broadly speaking, five ways for a user to prove his identity. A user can prove his identity through
```
Comment (yangc95) [inline content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md:21]:
```suggestion
There are, broadly speaking, five ways for a user to prove their identity. A user can prove their identity through the following:
```

File: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md [hunk 3]
```diff
@@ -14,21 +14,72 @@ CatalogContent:
...
 
-Other factors, depending on the verification system, include the following:
+- Something the user knows
+- Something the user has
+- Something the user is
+- Some*where* the user is
+- Some*when* the user is
```
Comment (yangc95) [inline content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md:27]:
```suggestion
- Something the user knows.
- Something the user has.
- Something the user is.
- Somewhere the user is.
- Somewhen the user is.
```

**Review by yangc95 — COMMENTED** [review]
@DCorneliusS Great edits and additions! I had a few comments and suggestions. Please tag me once you've taken all into consideration.

File: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md (hunk 2 above)
Comment (DCorneliusS) [reply content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md]: This changes have been added.

File: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md [hunk 4]
```diff
@@ -14,21 +14,72 @@ CatalogContent:
...
 ## Examples
 
-ATMs often require a physical card and a memorized PIN to successfully withdraw cash.
+### Something the User Knows
+
+Typically the first (and often only) factor used to verify user identity. Examples include the user knowing:
+
+- a password
+- a personal identification number (PIN)
+- a signature
```
[the remark this answers is not in the document]
Comment (DCorneliusS) [reply content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md]: This changes have been added.

File: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md [hunk 5]
```diff
@@ -14,21 +14,72 @@ CatalogContent:
...
+
+### Something the User Has
+
+This factor refers to a physical object in the possession of the user. Something the user has is often the second factor of authentication referred to in a 2FA authentication procedure. Examples include the user having:
+
+- a mobile phone
+- a USB key
+- a security token
+- a bank card
+- a physical key
```
[the remark this answers is not in the document]
Comment (DCorneliusS) [reply content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md]: These changes have been added.

File: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md [hunk 6]
```diff
@@ -14,21 +14,72 @@ CatalogContent:
...
+
+### Something the User Is
+
+Something the user is typically refers to the user's biology in the form of biometric markers. This factor can also refer to the user's behavioral tendencies. Examples include the user's:
+
+- fingerprint(s)
+- face
+- voice
+- iris(es)
+- keystroke typing dynamics
```
[the remark this answers is not in the document]
Comment (DCorneliusS) [reply content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md]: These changes have been added.

File: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md [hunk 7]
```diff
@@ -14,21 +14,72 @@ CatalogContent:
...
+
+### Somewhere the User Is
+
+This factor uses information about the user's location. Examples include the user's:
+
+- [geolocation](https://www.codecademy.com/resources/docs/general/geolocation)
+- proximity to a particular location
```
[the remark this answers is not in the document]
Comment (DCorneliusS) [reply content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md]: These changes have been added.

File: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md (hunk 3 above)
Comment (DCorneliusS) [reply content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md]: I have taken away the italicizing of the text, but I'd rather not add punctuation mark to the list items for the sake of consistency given that we have not added punctuation to the other listed items.

**Comment by DCorneliusS** [comment]
@yangc95 I have tried my best to take all of the feedback into account. Please see the two potentially unresolved conversations above to see if everything was taken care of adequately. 

Also, after updating my local branch, all I need to do is push it to my own local repo and the updates should be visible to everyone, yes? If that is not enough, then please feel free to suggest any further actions on my end.

Regards,

File: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md (hunk 1 above)
Comment (DCorneliusS) [reply content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md:17]: I am not sure I understand this request. Do you think it pertinent to add another sentence after line 17?

**Comment by SSwiniarski** [comment]
@DCorneliusS why did you close this PR? Are you withdrawing it?

File: content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md (hunk 1 above)
Comment (yangc95) [reply content/cybersecurity/concepts/multi-factor-authentication/multi-factor-authentication.md:17]: After a second look, I think this is straightforward and good as-is 🙂

**Comment by yangc95** [comment]
@DCorneliusS here's the live link: https://www.codecademy.com/resources/docs/cybersecurity/multi-factor-authentication

> It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
> 
> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> image
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> githublogin
>
> If the steps above do not work, it may be due to an email or local configuration issue.

@SSwiniarski

**Pull request merged**